src/views/ForgotPassword.tsx

```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import { Formik } from "formik";
+import * as yup from "yup";
+import axios from "axios";
+import { ToastContainer, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+import { Link } from "react-router-dom";
+import TextInput from "../components/TextInput";
+import { GrFormNextLink } from "react-icons/gr";
+
+const ForgotPassword: React.FC = () => {
+  const emailPattern = /^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/;
+
+  const validationSchema = yup.object().shape({
+    email: yup
+      .string()
+      .required("Email is required")
+      .matches(emailPattern, "Please enter a valid email"),
+  });
+
+  const handleForgotPassword = async (values: { email: string }) => {
+    try {
+      const response = await axios.post(
+        "https://e-commerce-bitcrafters-bn-1mpf.onrender.com/users/reset/link",
+        { email: values.email },
+      );
```

Comment:
```suggestion
  const axiosClient=useAxiosInstance()
  const handleForgotPassword = async (values: { email: string }) => {
    try {
      const response = await axiosClient.post(
        "/users/reset/link",
        { email: values.email },
      );
```
You can use the instance that @AngeloChristian1 have worked on

Reply:
Yes i can use it